### DISCUSSION TITLE
Jetbrains Plugin

### DISCUSSION THREAD

**Discussion by SimonBerens** [prisma/prisma#12274] [Q&A] [answered]
Hi all, just wondering if an official plugin for jetbrains ides is something on the roadmap. There are 2 plugins currently on the market and they are both awful compared to the prisma vscode plugin.

**Comment by austincrim** [comment, marked as answer]
Hey @SimonBerens 👋 ,

We have seen some interest for a Jetbrains plugin in the past, most of the latest info is in [this existing issue](https://github.com/prisma/prisma/issues/1788). Feel free to add a +1 to that issue as it will help us prioritize the feature going forward.

In short, we don't have the capacity to prioritize an official plugin right now but we are happy to assist the community where we can. If we see a lot more demand on the issue linked above, we could revisit this decision.

Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
